### DISCUSSION TITLE
Tutulan takımları bulma sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#104] [Zeka Sorusu]
Ayşe, Emin ve İhsan ayrı ayrı takımları tutuyorlar.

- Eğer İhsan Beşiktaşlıysa, Emin Fenerbahçeli.
- Eğer İhsan Fenerbahçeliyse, Emin Galatasaraylı.
- Eğer Emin Beşiktaşlı değilse, Ayşe Fenerbahçeli.
- Eğer Ayşe Galatasaraylıysa, İhsan Fenerbahçeli.

Ayşe, Emin ve İhsan ın tuttukları takımları bulun.

**Comment by Murkrow067** [comment]
İhsan Beşiktaşlı olamaz çünkü ilk önermeye dayanarak Emin'i Fenerbahçeli kabul etmiş oluruz ve 3. önermeye göre Ayşe de Fenerbahçeli olur ancak bu mümkün değildir. İhsan Fenerbahçeli dersek 2.  önermeye göre Emin Galatasaraylı olur ve 3. önermeye göre Ayşe de Fenerbahçeli çıkacağından bu ihtimal de elenir. Bu durumda İhsan Galatasaraylıdır. Sonrasında Emin ya Fenerbahçeli ya da Beşiktaşlı olmak zorundadır ancak Fenerbahçeli olursa 3. önermeye göre Ayşe ile takımları çakışacağından bu da mümkün olamaz. Böylece kalan tek bir ihtimal olur: İhsan Galatasaraylı, Emin Beşiktaşlı ve Ayşe Fenerbahçelidir.
